overlore/townhall/service.py

```diff
@@ -48,27 +47,38 @@ async def start() -> None:
     if TORII_WS is None:
         raise RuntimeError("Failure to provide WS url")
 
-    db = DatabaseHandler.instance().init()
+    DatabaseHandler.instance().init()
 
     signal.signal(signal.SIGINT, handle_sigint)
     gpt_interface = GptInterface.instance()
     gpt_interface.init(OPENAI_API_KEY, OPENAI_EMBEDDINGS_API_KEY)
 
     bound_handler = functools.partial(service, extra_argument={"mock": args.mock})
-    overlore_pulse = serve(bound_handler, args.address, SERVICE_WS_PORT)
+    serve(bound_handler, args.address, SERVICE_WS_PORT)
 
     print(f"great job, starting this service on port {SERVICE_WS_PORT}. everything is perfect from now on.")
-    # arbitrarily choose just the first realm to sub to events on
-    await asyncio.gather(
-        overlore_pulse,
-        torii_event_sub(TORII_WS, db.process_event, Subscriptions.COMBAT_OUTCOME_EVENT_EMITTED),
-        torii_event_sub(TORII_WS, db.process_event, Subscriptions.ORDER_ACCEPTED_EVENT_EMITTED),
-    )
+
+    conn = sqlean.connect(args.world_db)
+
+    print("sanity")
+
+    while True:
+        cursor = conn.cursor()
+        cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+        tables = cursor.fetchall()
+        print(tables)
+        await asyncio.sleep(10)
+    # await asyncio.gather(
+    # overlore_pulse,
+    # torii_event_sub(TORII_WS, db.process_event, Subscriptions.COMBAT_OUTCOME_EVENT_EMITTED),
+    # torii_event_sub(TORII_WS, db.process_event, Subscriptions.ORDER_ACCEPTED_EVENT_EMITTED),
+    # )
 
 
 # hardcode for now, when more mature we need some plumbing to read this off a config
 def main() -> None:
     asyncio.run(start())
+    print("wtf")
```

Comment:
added this to sanity check that the program was just exiting early when the service stuff was started from docker. its possible whats happening is that an error is being thrown, but it isn't being logged correctly.